pkg/component/extensions/operatingsystemconfig/operatingsystemconfig.go

```diff
@@ -1033,6 +1033,9 @@ func KeyV2(
 		if kubeReserved := kubeletConfiguration.KubeReserved; kubeReserved != nil {
 			data = append(data, fmt.Sprintf("%s-%s-%s-%s", kubeReserved.CPU, kubeReserved.Memory, kubeReserved.PID, kubeReserved.EphemeralStorage))
 		}
+		if eviction := kubeletConfiguration.EvictionHard; eviction != nil && eviction.MemoryAvailable != nil {
```

Comment:
pls update https://github.com/gardener/gardener/blob/master/docs/usage/shoot_updates.md#rolling-update-triggers

Reply:
I've updated the documentation and also included the other `EvictionHard` fields.